### PR TITLE
Add new image to Knative Serving.

### PR DESCRIPTION
**Pull request by markusthoemmes**
Knative Serving added a new image to its deployment.

### CODE REVIEW

**Comment by mgencur** [comment]
The image needs to be listed here: https://github.com/openshift/knative-serving/blob/master/Makefile#L5

File: ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ binary_build_commands: make install
 test_binary_build_commands: make test-install
 tests:
 - as: e2e
-  commands: "INTERNAL_REGISTRY=image-registry.openshift-image-registry.svc:5000 ENABLE_ADMISSION_WEBHOOKS=false make test-e2e"
+  commands: "make test-e2e"
```
Comment (matzew) [inline ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next.yaml]: :+1:

**Comment by markusthoemmes** [comment]
@mgencur it's here https://github.com/openshift/knative-serving/blob/release-next/Makefile#L5. On the branch this PR is for.

**Pull request merged**